views/datablog.handlebars

```diff
@@ -0,0 +1,330 @@
+<div class="translated-content">
+  {{> top_nav }}
+
+  <div class="container mt-5">
+    <div class="row">
+      <div class="col">
+	<head><title>Insights from FindTheMasks-US Data</title> 
```

Comment:
happy to update this if needed but you'll need to set this in the layout. It'll need to be passed through as a local (like "ogTitle") and then the static.handlebars file will need to use it or fallback to a default title.

Reply:
You're going to be able to do this sort of thing much more efficiently than I can. If we make a second article, I'll be able to be a copy-paste monkey the next go-round.